packages/synapse-interface/pages/state-managed-bridge/index.tsx

```diff
@@ -225,6 +228,17 @@ const StateManagedBridge = () => {
   const executeBridge = async () => {
     let pendingPopup: any
 
+    const currentTimestamp: number = getUnixTimeMinutesFromNow(0)
+    const bridgeQuoteTimestamp = convertUuidToUnix(bridgeQuote.id)
+    const timeDifference = calculateTimeBetween(
+      currentTimestamp,
+      bridgeQuoteTimestamp
+    )
+
+    if (timeDifference > quoteTimeout && !isLoading) {
+      await getAndSetBridgeQuote()
```

Comment:
Don't think we should do this here after `executeBridge` has been called, which implies user has agreed to quote at `t_0`. If the quote changes at `t_1`, then they won't know the quote's changed since it won't be obvious to them.

Example: User sees RFQ quote from chainA to chainB for 100 USDC -> 99 USDC (`t_0`). They click bridge but don't sign just yet. RFQ ChainB inventory adjusts. Some seconds pass. `getAndSetBridgeQuote()` is called. Now the new quote is 100 USDC -> 80 USDC on CCTP/SynapseBridge (`t_1`), but there's no clear indicator to the user that anything's changed (since they're expecting is to sign the txn in their wallet). Issue is mainly the user agreed to quote at `t_0` but not `t_1` even though they're signing for it.